twoinputs/windpower-solarangle.ipynb

```diff
@@ -52,6 +52,7 @@
     "\n",
     "# normalize\n",
     "norm = np.linalg.norm(windpower)\n",
+    "# norm = np.max(windpower)\n",
```

Comment:
How come this was added?

Reply:
I was testing a different method of "norm"ing the data.